### ISSUE TITLE
Wrong number of arguments (given 2, expected 1)

### ISSUE THREAD

**Issue by azizsayfiddin** [pjmp/bropages#8]
```
~ bro --trace find
/home/aljustiet/.local/share/gem/ruby/3.0.0/gems/json_pure-1.8.1/lib/json/common.rb:155:in `initialize': wrong number of arguments (given 2, expected 1) (Argum>
        from /home/aljustiet/.local/share/gem/ruby/3.0.0/gems/json_pure-1.8.1/lib/json/common.rb:155:in `new'
        from /home/aljustiet/.local/share/gem/ruby/3.0.0/gems/json_pure-1.8.1/lib/json/common.rb:155:in `parse'
        from /home/aljustiet/.local/share/gem/ruby/3.0.0/gems/bropages-0.1.0/lib/bro.rb:264:in `block (2 levels) in <top (required)>'
        from /home/aljustiet/.local/share/gem/ruby/3.0.0/gems/commander-4.1.5/lib/commander/command.rb:180:in `call'
        from /home/aljustiet/.local/share/gem/ruby/3.0.0/gems/commander-4.1.5/lib/commander/command.rb:155:in `run'
        from /home/aljustiet/.local/share/gem/ruby/3.0.0/gems/commander-4.1.5/lib/commander/runner.rb:402:in `run_active_command'
        from /home/aljustiet/.local/share/gem/ruby/3.0.0/gems/commander-4.1.5/lib/commander/runner.rb:78:in `run!'
        from /home/aljustiet/.local/share/gem/ruby/3.0.0/gems/commander-4.1.5/lib/commander/delegates.rb:7:in `run!'
        from /home/aljustiet/.local/share/gem/ruby/3.0.0/gems/commander-4.1.5/lib/commander/import.rb:10:in `block in <top (required)>'
```

**Comment by pombadev** [comment]
Different project, you likely meant https://rubygems.org/gems/bropages

**Comment by azizsayfiddin** [comment]
> Different project, you likely meant https://rubygems.org/gems/bropages

Are the they same?

**Comment by pombadev** [comment]
The github repo is located at https://github.com/hubsmoke/bro, keep in mind the owner has archived the repo.

**Comment by azizsayfiddin** [comment]
> The github repo is located at https://github.com/hubsmoke/bro, keep in mind the owner has archived the repo.

What does the --no-paging do?
When I run `bro -t base16 --no-paging none find`, I have this:
```
~ bro --no-paging less find
thread 'main' panicked at /home/aljustiet/.cargo/registry/src/index.crates.io-6f17d22bba15001f/clap_builder-4.5.2/src/parser/error.rs:32:9:
Mismatch between definition and access of `no-paging`. Could not downcast to TypeId { t: 324675245860759320943513204350442872190 }, need to downcast to TypeId { t: 42966343538335219590177265727833432740 }

note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[1]    1127371 IOT instruction (core dumped)  bro --no-paging less find
```

**Comment by azizsayfiddin** [comment]
Where is the config file located?

**Comment by pombadev** [comment]
That issue has been fixed in v1.5.0, make sure you have the latest version installed
